src/analysis/inner_product_space/projection.lean

```diff
@@ -6,6 +6,7 @@ Authors: Zhouhang Zhou, Frédéric Dupuis, Heather Macbeth
 import analysis.convex.basic
 import analysis.inner_product_space.symmetric
 import analysis.normed_space.is_R_or_C
+import analysis.inner_product_space.basic
```

Comment:
Something odd here, since the new import, `analysis.inner_product_space.basic`, is already being transitively imported through `analysis.inner_product_space.symmetric`. Don't hold up merging for this, though!

Reply:
Yes, there are probably a few of these: sometimes the algorithm made mistakes (because it can't follow `example`, @[ancestor], and maybe some other things as well), and I had to do some fixes by hand. Then I made mistakes when adding imports back in....  :-(

Reply:
Fixed now.